site/app/templates/admin/GradeOverride.twig

```diff
@@ -80,3 +71,4 @@
         source: {{ student_full|raw }}
     });
 </script>
+<script src="/js/grade-override.js"></script>
```

Comment:
This should be added in `GradeOverrideView.php` using `$this->core->getOutput()->addInternalCss('grade-override.js');`